data/channels.csv

```diff
@@ -1025,7 +1025,7 @@ AlAoulaLaayoune.ma,Al Aoula Laayoune,قناة الأولى,,SNRT,MA,,Rabat,c/MA,
 AlAoulaMaroc.ma,Al Aoula Maroc,قناة الأولى,,SNRT,MA,,Rabat,c/MA,ara;fra;spa,general,FALSE,1962-03-03,,,http://www.alaoula.ma/,https://i.imgur.com/Q2LJMj3.png
 AlAoulaMiddleEast.ma,Al Aoula Middle East,قناة الأولى,,SNRT,MA,,Rabat,c/MA,ara;fra;spa,general,FALSE,1962-03-03,,,http://www.alaoula.ma/,https://i.imgur.com/wWKFDWf.png
 AlAqsaTV.ps,Al-Aqsa TV,قناة الأقصى,,Hamas,PS,,Gaza,c/PS,ara,general,FALSE,2006-01-09,,,,https://i.imgur.com/Z2rfrQ8.png
-Alarabiya.ae,Alarabiya,العربية,,,AE,,Dubai,c/AE,ara,news,FALSE,,,,https://www.alarabiya.net/,https://i.imgur.com/NXFkYFj.png
+Alarabiya.sa,Alarabiya,العربية,,MBC Group,SA,,Riyadh,c/AE,ara,news,FALSE,2003-03-03,,,https://www.alarabiya.net/,https://i.imgur.com/NXFkYFj.png

```

Comment:
Yes... but no. The channel is indeed based in Dubai, even if MBC Group has it.

Reply:
Thanks for that: fixed.